fbw-common/src/systems/instruments/src/ND/ND.tsx

```diff
@@ -777,8 +777,9 @@ class TopMessages extends DisplayComponent<{ bus: EventBus; ndMode: Subscribable
     return (
       <>
         <Layer x={384} y={28}>
-          {/* TODO verify */}
-          <text class="Green FontIntermediate MiddleAlign">{this.approachMessageValue}</text>
+          <text class="Green FontIntermediate MiddleAlign" style="white-space: pre">
```

Comment:
This touches a common file. Does this also affect the A380X?

Reply:
Yes, A380 has the same alignment as A320.